index.css

```diff
@@ -0,0 +1,130 @@
+* { box-sizing: border-box;}
+Body{
+  background-attachment: fixed;
+  background-image: url('http://www.fleming.kyschools.us/userfiles/316/Clipart%20Images/math.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  margin:auto;
+}
+.calculator-title{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.calculator-calcBody{
+  box-shadow: 30px 30px 30px grey;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+  border: 1px black solid;
+  background-color: grey;
+  border-width: medium;
+  border-radius: 10px;
+  top: 20%;
+  left:40%;
+  width: 20%;
+  height: 75%;
+}
+
+.calculator-valueDisplay{
+  /*display: flex;*/
+  /*justify-content: center;*/
+  /*align-items: center;*/
+  font-family: 'Roboto', sans-serif;
```

Comment:
if you just put ` font-family: 'Roboto', sans-serif;` on the outer most element `.calculator` it will cascade down to all of the children.